ext/libclementine-tagreader/tagreader.h

```diff
@@ -95,6 +95,14 @@ class TagReader {
       TagLib::Ogg::XiphComment* vorbis_comments,
       const pb::tagreader::SongMetadata& song) const;
 
+  // Helpers for GuessArtistAndTitle()
+  void RemoveExtension(std::string*) const;
```

Comment:
These helpers don't need to be in the header, they can be only in the `.cpp` inside an anonymous namespace.